### ISSUE TITLE
Functionality point 3

### ISSUE THREAD

**Issue by baileyjarvis2814** [baileyjarvis2814/overlook#3]
Accessibility
You must be able to tab through your app and use it without a mouse
Your app must still be usable when tested with a colorblind extension
You must score as close to 100% as possible with the Lighthouse and WAVE accessibility audits. Be prepared to explain any accessibility audits your application is failing.
Your HTML must be written semantically and should use ARIA attributes for any UI elements that are not understood by the screen reader (ONLY if needed / appropriate)
